docs/unattend.xml

```diff
@@ -16,19 +16,22 @@
          <CEIPEnabled>0</CEIPEnabled>
       </component>
       <component xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="Microsoft-Windows-Shell-Setup" processorArchitecture="amd64" publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="nonSxS">
+         <!-- Windows host computer name -->
          <ComputerName>windows-host</ComputerName>
-         <ProductKey>My_Product_key</ProductKey>
+         <!-- Windows product key -->
+         <ProductKey>_WINDOWS_PRODUCT_KEY_</ProductKey>
       </component>
    </settings>
    <settings pass="oobeSystem">
       <component xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="Microsoft-Windows-Shell-Setup" processorArchitecture="amd64" publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="nonSxS">
          <AutoLogon>
+            <Enabled>true</Enabled>
+            <Username>Administrator</Username>
             <Password>
-               <Value>MyPassword</Value>
+               <!-- Administrator password -->
+               <Value>_ADMINISTRATOR_PASSWORD_</Value>
                <PlainText>true</PlainText>
             </Password>
-            <Enabled>true</Enabled>
-            <Username>Administrator</Username>
```

Comment:
You have made a lot of these sort of movements for aesthetic purposes from what I can see. Please do such changes in a separate PR in the future.